src/pages/ProfilePage/ChangePassword.tsx

```
@@ -0,0 +1,141 @@
+      <h2 className="text-lg font-semibold text-neutral-foreground">
+        Change Password
+      </h2>
+      <p className="mb-6 text-caption text-muted-foreground">
+        Create a new password for your account
+      </p>
+
+      <form onSubmit={handleSubmit} className="space-y-5">
+        <div className="flex flex-col gap-2">
+          <label className="text-body font-medium text-neutral-foreground">
+            Current Password
+          </label>
```

Biome (2.3.13)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)